util/dvsim/LintCfg.py

```diff
@@ -74,12 +74,12 @@ def gen_results_summary(self):
         colalign = ('center', ) * len(header)
         table = [header]
 
+        # Links to individual results are all relative w.r.t. this path.
+        summary_results_path = self.results_server_dir
         keys = self.totals.get_keys(self.report_severities)
         for cfg in self.cfgs:
-            results_page = f'{cfg.results_server_dir}/results.html'
-            results_page_url = results_page.replace(
-                cfg.results_server_prefix, cfg.results_server_url_prefix)
-            name_with_link = f'[{cfg.name.upper()}]({results_page_url})'
+            name_with_link = cfg._get_results_page_link(
+                relative_to=summary_results_path)
```

Comment:
The markdown link format is needed to linkify.

Reply:
Hm... the `_get_results_page_link` function is supposed to return a markdown link if the `--publish` switch is on. Do I understand correctly that this does not use the upload functionality of Dvsim to push to the GCS bucket?

Reply:
Ah never mind - I should have read that bug you linked first ;).